lib/missing_transaction_test.go

```diff
@@ -0,0 +1,110 @@
+package sebak
```

Comment:
I don't get the purpose of these test code.
What do you intend by this test code?